src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -255,6 +255,8 @@ private void setCommonFields(
     methodViewBuilder.stubName(namer.getStubName(context.getTargetInterface()));
     methodViewBuilder.settingsGetterName(namer.getSettingsFunctionName(context.getMethod()));
     methodViewBuilder.callableName(context.getNamer().getCallableName(context.getMethod()));
+    methodViewBuilder.isRequestStreaming(context.getMethod().getRequestStreaming());
+    methodViewBuilder.isResponseStreaming(context.getMethod().getResponseStreaming());
```

Comment:
If you use an enum for the streaming type, you wouldn't need to add each boolean as a separate attribute.